oaknational/Oak-Web-Application
# LESQ-790: Subject listing page query

## Description

Music year: 1989

- Updates subject listing query to utilise new MV
- Creates processLesson function and test file

## Acceptance Criteria:

- [ ]  Subject listing page should look the same 

## Issue(s)

Fixes #LESQ-790

## How to test

1. Go to https://deploy-preview-2442--oak-web-application.netlify.thenational.academy
2. Click on any subject listing page, should render the correct unit / listing counts

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
